### ISSUE TITLE
Layout broken

### ISSUE THREAD

**Issue by pogopaule** [tmux-python/tmuxp#737]
### Step 1: Provide a summary of your problem
Using the configuration from https://tmuxp.git-pull.com/examples.html#short-hand-inline I get the following tmux
![image](https://user-images.githubusercontent.com/576949/154834989-a3ac522d-3206-482e-b73d-66c4a37054e4.png)

### Step 2: Provide tmuxp details
```
-------------------------
environment:
	dist: Linux-5.4.0-100-generic-x86_64-with-glibc2.29
	arch: x86_64
	uname: Linux; pogopaule-ThinkPad-T550; 5.4.0-100-generic
	version: #113-Ubuntu SMP Thu Feb 3 18:43:29 UTC 2022
-------------------------
python version: 3.8.10 (default, Nov 26 2021, 20:14:08)  [GCC 9.3.0]
system PATH: Now using node v16.13.0 (npm v8.1.0)
/home/pogopaule/.yarn/bin:/home/pogopaule/bin:/usr/local/bin:/home/pogopaule/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/local/go/bin
tmux version: 3.0
libtmux version: 0.10.3
tmuxp version: 1.9.4
tmux path: /usr/bin/tmux
tmuxp path: /home/pogopaule/.local/lib/python3.8/site-packages/tmuxp/__init__.py
shell: /usr/bin/zsh
-------------------------
tmux sessions:

	no server running on /tmp/tmux-1000/default
tmux windows:

	no server running on /tmp/tmux-1000/default
tmux panes:

	no server running on /tmp/tmux-1000/default
tmux global options:

	no server running on /tmp/tmux-1000/default
tmux window options:

	no server running on /tmp/tmux-1000/default

```

#### Steps to reproduce:
  1. `pip3 install --user tmuxp`
  2. create yaml file with https://tmuxp.git-pull.com/examples.html#short-hand-inline as content
  3. run `tmuxp load config.yaml`

#### Observed Results:
See image above with broken layout and output in just one pane

#### Expected Results:
A layout with equal space for each of the three panes and output in each of them.

#### Relevant Code:
My configuration https://github.com/pogopaule/dotfiles/

**Comment by tony** [comment]
I get the same result.  it's scrunched together.

This needs to get looked at

**Comment by tony** [comment]
Upgrading this to p4-important @pogopaule 

FYI: I may close or merge this into other issues - depending on if there's an earlier one with the same root cause.  I'm not quite sure yet.  I will keep you apprised

**Comment by tony** [comment]
@pogopaule @tutuca

v1.13.1 is live with a fix, via @nvasilas from #793 

[PyPI](https://pypi.org/project/tmuxp/1.13.1/), [GitHub Tag](https://github.com/tmux-python/tmuxp/tree/v1.13.1)

```console
pip install tmuxp==1.13.1
```

Any better now?

P.S. You may need to kill your tmux server or try on a fresh one by passing something like `tmuxp ./workspacefile.yaml -L OtherTmuxServer`

**Comment by tony** [comment]
My next change at v1.13.2 will probably bring this issue back, if so ping me.
